alchemy-web/src/content/docs/providers/cloudflare/bucket.md

```diff
@@ -42,6 +42,30 @@ const publicBucket = await R2Bucket("public-assets", {
   name: "public-assets",
```

Comment:
Did you update the docs on clean?

Reply:
Ok we neve had any lol. Might be worth adding it.

Also change the title of in the frontmatter to `Bucket` instead of `R2Bucket`